src/entities/Proposal/routes.ts

```diff
@@ -759,10 +782,10 @@ async function getGrants() {
 
         const contract = {
           symbol,
-          balance: parseInt(balance, 10) / 10 ** decimals,
-          vestedAmount: parseInt(vestedAmount, 10) / 10 ** decimals,
-          releasableAmount: parseInt(releasableAmount, 10) / 10 ** decimals,
-          released: parseInt(released, 10) / 10 ** decimals,
+          balance: Math.round(parseInt(balance, 10) / 10 ** decimals),
```

Comment:
I would not specify the base in the `parseInt` function, by default it is set to 10 and the code looks cleaner. Just my point of view, what do you think?